package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ui-toolkit",
-  "version": "0.18.0",
+  "version": "0.19.0",
```

Comment:
no need to change this, this is updated automatically when you release. If you change this here I'm pretty sure it will skip a version.